docs/product/explore/profiling/transaction-vs-continuous-profiling.mdx

```diff
@@ -73,3 +75,13 @@ Note, that while the profiling mode can't be changed at runtime, it's fine for d
 ## Differences When Using Sentry
 
 The major difference between continuous profiling and transaction-based profiling when using Sentry, is that with continuous profiling you'll be able to visualize a flamegraph for your entire application. This means, that you'll be able to take a step back from the previous transaction-based view and look at your application's runtime as a whole, which makes it easier to prioritize the functions that are slowing down your entire application and not just one particular transaction.
+
+## Managing Usage
+
+The main difference between transaction and continuous profiling mode is when the profiler is started and stopped. In transaction-based profiling, the profiler is started and stopped based on transactions. In continuous profiling, the profiler is started and stopped by the developer.
+
+In continuous profiling mode, the profiler keeps running, which mean you can gain visibility into the parts of your application that you havent instrumented. This is especially useful if your instrumentation is incomplete, however it can also mean that if your application is idling for long periods of time, the profiling data you send and get billed for, might not be very useful.
```

Comment:
```suggestion

The main difference between transaction-based and continuous profiling is when the profiler runs. Transaction-based profiling starts and stops with every transaction, while continuous profiling runs continuously, but can be manually started and stopped.

Continuous profiling can give you visibility into parts of your application that you haven't instrumented. This is especially useful if your instrumentation is incomplete. However, it may also capture idle time, resulting in data that's potentially less valuable.
```
I don't think we have to explicitly mention billing. I think people can understand the implication of paying for valuable vs less valuable data :)